### ISSUE TITLE
Issue while using clingo with asprin

### ISSUE THREAD

**Issue by PaoloPangallo** [potassco/clingo#516]
I have clingo 5.7.1 and asprin 3.1.0 and it seems that I can t run asprin proprerly 
![image](https://github.com/user-attachments/assets/f8ee8da5-b074-4d85-ad4e-061c8ec969b8)

the same error shows up when i try to run the followingcode on pycharm :

import sys
print("Python version:")
print(sys.version)

try:
    import clingo
    print("Clingo imported successfully")
    print("Clingo version:")
    print(clingo.version)
except ImportError as e:
    print("Failed to import clingo:", e)

import subprocess

def run_asprin():
    try:
        result = [subprocess.run](https://subprocess.run/)(['asprin', '--version'], capture_output=True, text=True, check=True)
        print("Asprin ok, output:")
        print(result.stdout)
    except subprocess.CalledProcessError as e:
        print("Error:", e)
        print("stdout:", e.stdout)
        print("stderr:", e.stderr)
    except Exception as e:
        print("Unexpected error", e)

#
run_asprin()

![image](https://github.com/user-attachments/assets/f4e394e7-df67-447f-81b3-9c6a76db1fe6)

**Comment by rkaminsk** [comment]
I don't think this has anything to do with clingo. It might be that you have to use an older version. I'll transfer the issue to the asprin repo.
